relayer/crates/starknet-chain-components/src/impls/queries/packet_commitment.rs

```diff
@@ -85,24 +91,21 @@ where
 
         let block = chain.query_block(height).await?;
 
-        let unsigned_membership_proof_bytes = MembershipVerifierContainer {
-            state_root: block.block_hash.to_bytes_be().to_vec(),
-            prefix: chain.ibc_commitment_prefix().clone(),
-            path: Path::Commitment(CommitmentPath::new(port_id, channel_id, *sequence))
-                .to_string()
-                .into(),
-            value: commitment_bytes.clone(),
-        }
-        .canonical_bytes();
-
-        let signed_bytes = chain
-            .proof_signer()
-            .sign(&unsigned_membership_proof_bytes)
-            .map_err(Chain::raise_error)?;
+        let ibc_path = Path::Commitment(CommitmentPath::new(port_id, channel_id, *sequence));
+
+        let felt_path: Felt = ibc_path_to_storage_key(&StarknetCryptoLib, ibc_path);
+
+        // key == path
+        let storage_proof: StorageProof = chain
+            .query_storage_proof(height, &contract_address, &[felt_path])
+            .await
+            .unwrap();
+
+        let storage_proof_bytes = serde_json::to_vec(&storage_proof).unwrap();
```

Comment:
```suggestion
        let storage_proof_bytes = serde_json::to_vec(&storage_proof).map_err(Chain::raise_error)?;
```
This might require adding a CanRaiseAsyncError